File: csharp/darts/Darts.cs
using System;
using System.Collections.Generic;

public static class Darts
{
    public const double OUTER_RADIUS = 10; 
    public const double MIDDLE_RADIUS = 5;
    public const double INNER_RADIUS = 1; 
    public const int NO_TARGET_SCORE = 0;
    public const int OUTER_RADIUS_SCORE = 1;
    public const int MIDDLE_RADIUS_SCORE = 5;
    public const int INNER_RADIUS_SCORE = 10; 


    public static int Score(double x, double y)
    {
        double distance = Math.Sqrt(x*x + y*y);

        if(distance <= INNER_RADIUS)
            return INNER_RADIUS_SCORE;
        else if(distance <= MIDDLE_RADIUS)
            return MIDDLE_RADIUS_SCORE;
        else if(distance <= OUTER_RADIUS)
            return OUTER_RADIUS_SCORE;
        else
            return NO_TARGET_SCORE;
       
    }
}

File: python/darts/darts.py
 
def score(x, y):
    outer_radius = 10
    middle_radius = 5
    inner_radius = 1 
    no_target_score = 0
    outer_radius_score = 1
    middle_radius_score = 5
    inner_radius_score = 10

    distance = pow(x*x+y*y, 0.5) 

    if distance <= inner_radius:
        return inner_radius_score
    elif distance <= middle_radius:
        return middle_radius_score
    elif distance <= outer_radius:
        return outer_radius_score
    else:
        return no_target_score
